package.json

```
@@ -15,5 +15,5 @@
     "postversion": "git push --follow-tags",
     "sync": "git checkout master && git pull && git fetch upstream && git merge upstream/master && git push origin --follow-tags"
   },
-  "version": "0.31.4-0"
+  "version": "0.31.5-0"
```

```suggestion
  "version": "0.31.4-1"
```
sorry it should be this actually, since there's already a prepatch for this (or else we'll stray from the upstream)